addons/website_livechat/static/src/components/discuss/discuss.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Discuss.content" t-inherit-mode="extension">
+        <xpath expr="//ThreadView" position="before">
```

Comment:
prefer class instead of type of element

Reply:
Shouldn't it be inside the `ThreadView` itself actually? I'm pretty sure the banner was there on chat window too, not only discuss.

Reply:
After testing 13.3, in chatter it's not this banner though, but I remember a blue banner similar to out of office at some point. Maybe it was introduced in a commit between 13.3 and our merge?

Reply:
I don't know, @alexkuhn knows ?

Reply:
I can't find what it was even with 1 commit before, maybe I'm thinking about something else... Just change to use class name in xpath and forget about the rest here.

Reply:
Out of office banner is something else, was done in 12.1 or so, not sure if we still have it in 13.4 or not (and if we haven't, whether it's an omission from us or removal or feature, I dunno).

Here it's about livechat banner, something introduced in 12.5 by DBE I think. In 13.3 when you have livechat, it's a menu on top of conversation in discuss app.

I have some ideas of what would eventually be the design for this topbar, which has indeed something to do to being on all `ThreadView`... But that's definitely out of scope of PR, especially because it's not 100% what we want and it's surely a lot of effort to do (moving most fields from `mail.chatter` to `mail.thread` is a hard-requirement). So please limit to discuss app, like in 13.3, with very few amount of changes in code. Ignore support of this in other views such as chat window, this will eventually become a thing at a much later date.